app/app.go

```diff
@@ -680,6 +681,22 @@ func newETH2Client(ctx context.Context, conf Config, life *lifecycle.Manager,
 		eth2Cl = eth2wrap.WithSyntheticDuties(eth2Cl, pubkeys)
 	}
 
+	// Check BN chain/network.
```

Comment:
can move this BN check to before eth2Cl is initialized

Reply:
mmmm, you need the eth2Cl for the check, so this isn't possible?